.github/dependabot.yml

```diff
@@ -0,0 +1,15 @@
+version: 2
+updates:
+  - package-ecosystem: npm
+    directory: /
+    schedule:
+      interval: monthly
+    allow:
+      - dependency-type: direct
+    ignore:
+      - dependency-name: stylelint
+        versions: [^15]
```

Comment:
Correct syntax update for versions.

Reply:
When I ran npm install, I was getting deprecated warnings from stylelint, and the warnings were resolved when latest version of stylelint is installed. I am wondering, Is there reason why we are ignoring stylelint update from dependabot?

Reply:
See the original PR where I added the dependency restriction: https://github.com/fluid-project/trivet/pull/357. The short answer is that `stylelint-config-fluid` is currently incompatible with newer versions of stylelint.